src/Routes/RosPage/RosPage.js

```diff
@@ -310,9 +329,30 @@ class RosPage extends React.Component {
                                     }
                                 ]
                             }}
+                            exportConfig={{
+                                extraItems: [<Button
+                                    key='pdf-download-button' variant='plain'
+                                    onClick={() => this.setExportSystemsPDF(true)}>
+                                        Export as PDF
+                                </Button>],
+                                ouiaId: 'export',
+                                onSelect: (_event, fileType) => this.onExportOptionSelect(fileType)
+                            }}
                             onExpandClick={(_e, _i, isOpen, { id }) => this.props.expandRow(id, isOpen, 'EXPAND_ROW')}
                         >
                         </InventoryTable>
+                        {exportSystemsPDF &&
+                            <DownloadSystemsPDFReport
+                                showButton={false}
+                                onSuccess={() => this.setExportSystemsPDF(false)}
+                                filters={{
+                                    stateFilter: stateFilterValue,
+                                    hostnameOrId: nameFilterValue
```

Comment:
On using `name` filter and not entering full name of a system in the filter, this will generate the pdf without containing the full name at the top of the report. Should we consider adding full name?

![Screenshot from 2022-03-28 10-03-43](https://user-images.githubusercontent.com/61837065/160327328-2c3abf17-f4b1-421a-b806-17b89b87c24a.png)

Reply:
good question @r14chandra since it may happen that there might be many systems based on the name search hence I think it will be good to only include search text rather than full name for now as I also checked Vulnerability service and their PDF report only have search term in case of `Name` filter. Hope this helps!